packages/wrangler/src/config/environment.ts

```diff
@@ -325,6 +325,44 @@ interface EnvironmentNonInheritable {
 		preview_id?: string;
 	}[];
 
+	/**
+	 * Specifies Queues that are bound to this Worker environment.
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `{}`
+	 * @nonInheritable
+	 */
+	queues: {
+		/** Producer bindings */
+		producers: {
+			/** The binding name used to refer to the Queue in the worker. */
+			binding: string;
+
+			/** The name of this Queue. */
+			queue: string;
+		}[];
+
+		/** Consumer configuration */
+		consumers: {
+			/** The name of the queue from which this script should consume. */
+			queue: string;
+
+			/** The maximum number of messages per batch */
+			batch_size?: number;
```

Comment:
```suggestion
			max_batch_size?: number;
```

Reply:
There was some confusion when we talked about the consumer settings, having `max` makes it more explicit.